docs/source/demo.rst

```diff
@@ -95,7 +95,7 @@ equivalent command using:
 .. code-block:: bash
 
     $ source ga4gh-env/bin/activate
-    (ga4gh-env) $ ga4gh_client readgroupsets-search --datasetIds XXXX http://localhost:8000/current
+    (ga4gh-env) $ ga4gh_client readgroupsets-search --dataset_ids XXXX http://localhost:8000/current
```

Comment:
Sorry, for confusing things. There are two points here I guess:

1) Does `readgroupsets-search --dataset_ids XXXX` work on this branch? I was concerned that argparse might have automatically changed these underscores to hyphens.
2) Should we change `--dataset_ids` to ` --dataset-ids` for POSIX compliance?

I think the answer to (2) is a clear, "yes, but later". I'm still not sure about (1) though. Do these examples work on your branch @hjellinek?